### ISSUE TITLE
Scrollbar "slides" out from under you as you drag the scroll box

### ISSUE THREAD

**Issue by TonyGermaneri** [TonyGermaneri/canvas-datagrid#97]
### Expected behavior and actual behavior.

The scroll box should stay exactly below the point where the mouse/touch picked it up.

Right now the scroll box "slides" ahead or behind the mouse/touch.

### Steps to reproduce the problem.

Get a large scroll box (lots of data) click and drag, watch the box slide out from under your cursor the further away from the point of origin you get.

### Specifications like the version of the project, operating system, or hardware.

> v0.17.x

**Comment by TonyGermaneri** [comment]
This is the source of the defect.  I believe these offset numbers need to be a result of calculations made to the total size of the grid.  The behavior changes ("sliding") when the grid is different sizes and these numbers are different.
https://github.com/TonyGermaneri/canvas-datagrid/blob/master/lib/events.js#L122

**Comment by TonyGermaneri** [comment]
I will give anyone who can fix this bug $200 USD.  I believe this is the last major bug that prevents the release of v1.  I can't fix it because I am a stupid man.  I will continue to try.

**Comment by bivald** [comment]
Is this the same issue as the scroll dragging "stops" if the mouse moves outside of the canvas? See screen recording at https://www.dropbox.com/s/0uh0b9g20163gcu/datagrid.mov?dl=0

**Comment by TonyGermaneri** [comment]
No, that's a bug that was introduced by a fix for "the mouse never lets go when I move off the browser".  I need to add another fix for the fix.  Thankfully that's an easy fix.
